lib/diagnostic.rb

```diff
@@ -21,20 +27,26 @@ def sum(enum)
   # in the numbers enumerable are odd
   def some_odd(numbers)
     # Your code goes here
+   numbers.one? { |num| num.odd? }
   end
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
   def every_even(numbers)
-    # Your code goes here
+    evens = numbers.all { |num| num.even? }
+    puts evens
   end
 
   # Question 5
   # Write ruby code that returns an array with all of the
   # entries from the words enumerable capitalized (i.e. 'foo' becomes 'Foo').
   def transform(words)
-    # Your code goes here
+    lines = filename.open(ARGV[0]) do |file|
+      file.map(&:itself).capitalize
+    end
+
+    print lines
```

Comment:
No, we're not opening a file in this example. We were just looking for a simple `.map` here.
e.g. `words.map(&:capitalize)`